samples/SampleStartups/SampleStartups.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
```

Comment:
No. This is a class library. RID is only required for exe projects, and as a workaround for https://github.com/dotnet/sdk/issues/396